wiki/resources/tools/discord.md

```diff
@@ -182,6 +182,12 @@ description: Discord related tools
 [Server Invite](https://discord.gg/tbzP3XaUF7)   <br/>
 **Credit:** @537875750955778058
 
+## Avatars
+
+> **Description:** A website to download thousands of Discord Pfps. Free!   <br/>
+**Link:**
+[PfpFinder](https://pfpfinder.com)   <br/>
```

Comment:
That site seems to be a pfps.gg clone bloated with ads lets use https://pfps.gg/

Reply:
Aditionally if we add a pfp source lets add oke for banners and colors aswell in the same commit. So please update your title to be actually descriptive!